boot.nix

```diff
@@ -1,7 +1,6 @@
 { system ? null
 , patches ? [ ]
 , extraOverlays ? [ ]
-, overlays ? [ (import ./.) ]
```

Comment:
I don't think this was ever used? And the overlays is the whole point anyway right?

Reply:
I thought this was here if wanted to add more than one overlay to our fixed set of packages.

Reply:
But then you have `extraOverlays` right?

Reply:
I guess. I forget why it's there. maybe fine to keep like this.